### ISSUE TITLE
Add option to manually run workflow action

### ISSUE THREAD

**Issue by sparr** [irgolic/AutoPR#23]
Create a second github workflow triggered on the `workflow_dispatch` event. Configure this event to accept inputs specifying the issue from which to take input, so that contributors to this repo can trigger the workflow to run on issues created by non-contributors. Refactor the existing `create-pr-from-issue` workflow and move functionality common between that workflow and this new workflow to another new callable workflow that they can both call.

**Comment by irgolic** [comment]
I don't quite understand how the `workflow_dispatch` event works, but I ran this through AutoPR anyway.

Note, Github explicitly doesn't allow the actions user (using `secrets.GITHUB_TOKEN`) to touch the workflows directory. So I ran it with my personal access token.

It seems like it didn't correctly parse the diffs for the last two commits. I'm still working on improvements for diff parsing, at the moment it's just [one big vertical extractor](https://github.com/irgolic/AutoPR/blob/main/autopr/validators.py#L231).

Thanks for taking your time to try out AutoPR :)
